README.md

```diff
@@ -1,2 +1,19 @@
 # odin-recipes
+
 Recipes Website
+
+![screenshot](/images/site_screenshot.png)
+
+Recipes website for learning to code from the odin project using only basic html
+
+## Demo
+
+https://cerdtruir.github.io/odin-recipes/
+
+## Built With
+
+HTML5
+
```

Comment:
An `acknowledgements` section, referencing (in this case) the Odin project, would be a nice touch, as well as an `author` section. Here is an [example](https://github.com/ikraamg/Bootstrap-NewsWeek-Replica), but there is no need to use every section.